frontend/cypress/pages/DashBoard.js

```diff
@@ -1,81 +1,131 @@
 class DashBoardPage {
-  addOrder(Program) {
-    cy.fixture("Order").then((order) => {
```

Comment:
I think this was a good generic function to add an order with a program.
Can you just improve this function selectors and re-use it across the three programs??